addons/mail/static/src/components/activity/activity.xml

```diff
@@ -47,7 +47,7 @@
                                 <t t-esc="assignedUserText"/>
                             </div>
                         </t>
-                        <a class="o_Activity_detailsButton btn btn-link" t-on-click="_onClickDetailsButton" role="button">
+                        <a href="javascript:;" class="o_Activity_detailsButton btn btn-link" t-on-click="_onClickDetailsButton" role="button" t-att-aria-expanded="state.areDetailsVisible ? 'true' : 'false'">
```

Comment:
Maybe make it a button instead? Don't like dummy href. If not possible, at least have it to # and prevent default instead.

Reply:
Even though button is the ideal solution, it requires CSS to don't change current appearance. I tried it below but, as you pointed out on https://github.com/odoo/odoo/pull/86324#discussion_r834140502, it's extra code to maintain that shouldn't be required.

Followed the `href="#"` and prevent default approach.